packages/remix-cloudflare/package.json

```diff
@@ -21,7 +21,9 @@
     "@cloudflare/workers-types": "^3.4.0"
   },
   "peerDependencies": {
-    "@cloudflare/workers-types": "^2.0.0 || ^3.0.0"
+    "@cloudflare/workers-types": "^2.0.0 || ^3.0.0",
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
```

Comment:
We already had a couple of PRs doing this and we decided we don't want this as these aren't necessary in any way.
This is only there because we're relying on `react-router`, which has these in `peerDependencies` as well.

Once we're switch to `@remix-run/router` (once `react-router` v6.4 is released), these won't be necessary anymore in `peerDependencies`, so all problems will be gone as well.

```suggestion
    "@cloudflare/workers-types": "^2.0.0 || ^3.0.0"
```

Reply:
Thank you for that context. The desired result (no warning when using Yarn) can also be achieved by setting 
```json
  "peerDependenciesMeta": {
    "react": {
      "optional": true
    },
    "react-dom": {
      "optional": true
    }
  }
```

in the `package.json` of `remix-server-runtime`. I’ve made this change now, let me know if you disagree

Reply:
Since it's only a warning, I think we can just leave it as-is
The warning will disappear automatically once we made the switch to `@remix-run/router`.

Reply:
While this might be fixed in the future, I’d still err on the side of correctness in the meantime.

This is a warning when running `yarn install`, but an error (via `@yarnpkg/esbuild-plugin-pnp` when running `yarn build`):
<img width="701" alt="Screenshot 2022-06-25 at 17 41 40" src="https://user-images.githubusercontent.com/20876627/175780799-8bee29bc-4014-48af-81dc-0462974d1068.png">